src/Redux/Missions/missions.js

```diff
@@ -0,0 +1,57 @@
+import { createSlice } from '@reduxjs/toolkit';
+import getMissionsData from './missionsAPI';
+
+const initialState = {
```

Comment:
- [ ] Rockets is the default view, so its data would be fetched when the application starts. However, the missions data should only be fetched (once) when a user navigates to the Missions section, kindly modify your code so the mission data will not be fetched until the user clicks on the `mission` link

Reply:
Fixed